doc/project/release_process.rst

```diff
@@ -85,24 +85,146 @@ At that point, the whole process starts over again.
 
 .. _release_quality_criteria:
 
-Release Quality Criteria
-************************
+Release Criteria
+****************
+
+The main motivation is to clearly have the criteria in place that must be met for a public releases to ship.
+This will help define when a release is "done" in terms that most people can
+understand and in ways that help new people to understand the process and
+participate in creating successful releases:
+
+- The release criteria documents all the requirements of our target audience for each Zephyr release
+- The target audiences for each public release can be different, and may overlap
+- The criteria at any given time are not set in stone: there may be requirements
+  that have been overlooked, or that are new, and in these cases, the criteria
+  should be expanded to ensure all needs are covered.
+
+Below is the high level criteria to be met for each release:
+
+- No blocker bugs
+- All relevant tests shall pass on Tier 0 platforms
+- A test subset shall pass on Tiers 0 and 1 (at least 1 per architecture/architecture variant/Hardware features)
+- All samples/tests shall build on Tiers 0, 1 and 2
+- All high and critical static analysis and security issues addressed
+- Footprint/Performance Tracking (not specific KPIs)
+- License audits pass
+- Documentation Completeness and Release Notes
+- Per Subsystem release criteria + Testing
```

Comment:
I would think that it's already implicitly assumed that the release criteria apply to all subsystems, so maybe drop it?